### ISSUE TITLE
c5_m5_checks_script.sh falsely complaining about NVMe on Ubuntu 18.04 (Bionic)

### ISSUE THREAD

**Issue by netsnipe** [awslabs/aws-support-tools#86]
The -aws kernel in Ubuntu 18.04 is no longer compiling NVMe support as a separate kernel module, so c5_m5_checks_script.sh now provides inaccurate advice.

```
$ sudo ./c5_m5_checks_script.sh
-e ------------------------------------------------
ERROR  NVMe Module is not available on your instance.
	- Please install NVMe module before changing your instance type to M5/C5. Look at the following link for further guidance:
-e 	> https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html
-e

OK     ENA Module with version 2.0.3K is installed and available on your instance
-e

OK     fstab file looks fine and does not contain any device names.
-e
------------------------------------------------

$ uname -r
4.15.0-1039-aws

$ dmesg | grep -i nvme
[    0.000000] Command line: BOOT_IMAGE=/boot/vmlinuz-4.15.0-1039-aws root=UUID=bbf64c6d-bc15-4ae0-aa4c-608fd9820d95 ro console=tty1 console=ttyS0 nvme.io_timeout=4294967295
[    0.000000] Kernel command line: BOOT_IMAGE=/boot/vmlinuz-4.15.0-1039-aws root=UUID=bbf64c6d-bc15-4ae0-aa4c-608fd9820d95 ro console=tty1 console=ttyS0 nvme.io_timeout=4294967295
[    0.931705] nvme nvme0: pci function 0000:00:04.0
[    2.260142]  nvme0n1: p1
[    7.315067] EXT4-fs (nvme0n1p1): mounted filesystem with ordered data mode. Opts: (null)
[    7.910372] EXT4-fs (nvme0n1p1): re-mounted. Opts: discard

$ sudo lsmod | grep -i nvme

$ grep -i NVME /boot/config-4.15.0-1039-aws
# NVME Support
CONFIG_NVME_CORE=y
...
```

**Comment by lpierillas** [comment]
Hi,

Thank you for the feedback. I see there is a pull request opened for this already so we'll review it to see if it would fix the issue.

**Comment by lpierillas** [comment]
Should be fixed by #92
